tests/main/api-get-systems-label/task.yaml

```diff
@@ -0,0 +1,22 @@
+summary: verify that the /systems/<label> api works
+
+systems:
+  # TODO: also test classic/core hybrid systems once they are ready
+  - ubuntu-core-2*
+
+execute: |
+  snap install --edge jq
+  # devmode as the snap does not have snapd-control
+  snap install test-snapd-curl --devmode --edge
+
+  echo "Find what systems are available"
+  test-snapd-curl.curl -s --unix-socket /run/snapd.socket http://localhost/v2/systems > systems
+  current_label=$(jq -r '.result.systems[0]["label"]' < systems)
+
+  echo "Get details for a specific system"
+  test-snapd-curl.curl -s --unix-socket /run/snapd.socket http://localhost/v2/systems/"$current_label" > current-system
+  echo "Ensure the result contains a model assertion"
+  jq -r '.result.model.type' < current-system | MATCH model
```

Comment:
here we should also check the various things that we expect because of the cliemt.System compatibility